mongodb_exporter.go

```diff
@@ -62,6 +63,9 @@ var (
 	maxConnectionsF               = flag.Int("mongodb.max-connections", 1, "Max number of pooled connections to the database.")
 	testF                         = flag.Bool("test", false, "Check MongoDB connection, print buildInfo() information and exit.")
 
+	socketTimeoutF = flag.Int64("mongodb.socket-timeout", int64(3*time.Second), "Amount of time in ms to wait for a non-responding socket to the database before it is forcefully closed")
+	syncTimeoutF   = flag.Int64("mongodb.sync-timeout", int64(1*time.Minute), "Amount of time in ms an operation with this session will wait before returning an error in case a connection to a usable server can't be established")
```

Comment:
There is [flag.Duration](https://golang.org/pkg/flag/#Duration) for this purpose.
Example https://github.com/percona/mysqld_exporter/blob/50377ef0863147751e3057fa95b263ed396ad2f4/mysqld_exporter.go#L82

Reply:
Good idea!